lib/Service.php

```diff
@@ -230,6 +230,33 @@ function mapValueObject($elementName, $className) {
         $this->classMap[$className] = function(Writer $writer, $valueObject) use ($namespace) {
             return \Sabre\Xml\Serializer\valueObject($writer, $valueObject, $namespace);
         };
+        $this->valueObjectMap[$className] = $elementName;
```

Comment:
Note related to this PR: do we need to make sure (or help preventing users people shooting in there feet)?

- check whether a classMapping already exists before mapping the value object
- check whether the elementMapping already exists before kaüping the value object